GliaWidgets/Sources/ViewModel/Chat/Data/OutgoingMessage.swift

```diff
@@ -3,18 +3,30 @@ import GliaCoreSDK
 
 class OutgoingMessage: Equatable {
     let files: [LocalFile]
+    let relation: Relation
```

Comment:
It would be nice to add some documentation to this field.

Reply:
Added 7aafbc34102a837fae6359bc4b05161ef4049bf0